src/lib/models/events-group.ts

```diff
@@ -196,6 +220,18 @@ export class EventGroups {
       const { scheduledEventId } = event.activityTaskCompletedEventAttributes;
       this.get(scheduledEventId).set(event.eventType, event);
     }
+
+    if (event.eventType === 'ChildWorkflowExecutionStarted') {
```

Comment:
Eventually we're going to need to go back and address the test coverage. I think that we should ship the functionality for now, but we'll need to pay down the technical debt over the next few weeks.